ocaml/src/nsm_model.ml

```diff
@@ -1598,7 +1579,9 @@ module NamespaceManager(C : Constants)(KV : Read_key_value_store) = struct
       Update'.compare_and_swap
         (Keys.objects ~object_id)
         None
-        (Some (serialize Manifest.output manifest)) in
+        (Some (serialize (Manifest.to_buffer
+                            ~version:1 (* TODO: should become 2 *)) manifest))
```

Comment:
We also might want to bulk change stored manifests as well at some point. I'll leave the TODO.